libs/template/application/src/app/app-builder.ts

```diff
@@ -0,0 +1,6 @@
+import { AppBuilderWithModules } from './app.model';
+import { ModularAppBuilder } from './modular-app-builder';
+
+export function appBuilder(): AppBuilderWithModules {
+  return new ModularAppBuilder();
```

Comment:
I'm not sure why do we need 2 separate appBuilders in core and application packages?
We had only one before, what's the reason to have 2 now?